system_test/aest_sync_SUITE.erl

```diff
@@ -410,17 +408,3 @@ net_split_recovery(Cfg) ->
     ?assertEqual(D1, D4),
 
     ok.
-
-%=== INTERNAL FUNCTIONS ========================================================
-
-get_block(NodeName, Height, _Cfg) ->
```

Comment:
I understand that the manager is now a singleton, but the API wasn't exposing this by passing the context around. I don't like much the fact that some helper functions assume it is a singleton and the other don't.

Reply:
I agree, and this is a first attempt at a cleanup. The tests in `aest_sync_SUITE` already used the singleton. I wanted to simplify the API for now and remove some unused code.